static/js/load_map.js

```diff
@@ -75,4 +76,60 @@ function initialize() {
             markers.push(nextMarker);
         });
     });
+
+    const stores = document.getElementById('Stores');
+    function getLocation() {
+        if (navigator.geolocation) {
+            return navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            x.innerHTML = "Geolocation is not supported by this browser.";
+        }
+    }
+
+    function showPosition(position) {
+        x.innerHTML = "Latitude: " + position.coords.latitude +
+        "<br>Longitude: " + position.coords.longitude;
+    }
+
+    function callback(results, status) {
+      if (status == google.maps.places.PlacesServiceStatus.OK){
+        markers.forEach(marker => {
+            marker.setMap(null);
+        });
```

Comment:
Is this considered good javascript style?

IMO, you should use normal `for` loops for stateful changes (like this) and only use functional things like `forEach` or `map` for data transformations. But I could be wrong here.